docs/howto/operator/gcp/sql.rst

```diff
@@ -639,5 +640,5 @@ Templating
 More information
 """"""""""""""""
 
-See `Google Cloud SQL Proxy documentation
+See `Google Cloud SQL documentation for more information
```

Comment:
Same here

Reply:
adding links for mysql and postgres documentation